src/components/ThumbnailsPanel/ThumbnailsPanel.js

```diff
@@ -35,7 +35,7 @@ class ThumbnailsPanel extends React.PureComponent {
     this.listRef = React.createRef();
     this.afterMovePageNumber = null;
     this.state = {
-      numberOfColumns: this.getNumberOfColumns(),
+      numberOfColumns: 1,
```

Comment:
Hmm after another look it seems to me that it's not necessary to store `numberOfColumns` in `this.state` anymore. Since we can just calculate this value from `width`.

Can we remove this from state? Sorry for not finding this out in the first place...

Reply:
removed